README.md

```diff
@@ -13,6 +13,7 @@
 <img src="./widescreen.png" width="100%" height="auto"/>
 
 It is still a working project but you can watch a live demo here:  
+
 https://vicperalta.github.io/Portfolio/
 
```

Comment:
- [ ] **Required:** You have added the link as stated, however, you haven't used the appropriate markdown link styling for your link. Example `[Liive Demo](https://live.demo)`.

Reply:
Thanks